Libplanet.Net/BoundPeer.cs

```diff
@@ -103,6 +90,17 @@ private BoundPeer(SerializationInfo info, StreamingContext context)
 
         public string PeerString => $"{PublicKey},{EndPoint.Host},{EndPoint.Port}";
 
+        /// <inheritdoc/>
+        [Pure]
+        public Bencodex.Types.IValue Bencoded =>
+            Bencodex.Types.Dictionary.Empty
+                .Add(PublicKeyKey, PublicKey.Format(true))
+                .Add(EndPointHostKey, EndPoint.Host)
+                .Add(EndPointPortKey, EndPoint.Port)
+                .Add(
+                    PublicIpAddressKey,
+                    PublicIPAddress is IPAddress ip ? (IValue)(Text)ip.ToString() : Null.Value);
+
```

Comment:
A question: we currently have objects that are encoded to bencodex values with interfaces like `IValue Serialize()` (`Dictionary ToBencodex()` too, but I see that this is being changed too.) Is it intended that objects that can be represented to bencodex values all use the `IBencodable` interface and have those different APIs changed into `Bencoded`?

Reply:
Yup, trying to have some sort of standard for housekeeping. As you've mentioned, there are two issues:

- Method names for "serializing" to bencodex are not standardized.
- Return types are not standardized.

Perhaps the decision to not return a concrete type might cause some issues down the line, but we'll deal with it later. 😅